frontend/lib/search/results_loader.dart

```diff
@@ -8,6 +8,8 @@ import 'package:infinite_scroll_pagination/infinite_scroll_pagination.dart';
 
 import 'package:ehrenamtskarte/l10n/translations.g.dart';
 
+import '../home/home_page.dart';
```

Comment:
please use package import

Reply:
Is there no linter for that :o

Reply:
I went ahead and enabled it and fixed a few more occurences